Assets/MixedRealityToolkit/_Core/Utilities/UX/HoverLight.cs

```diff
@@ -13,46 +13,32 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Utilities.UX
     [ExecuteInEditMode]
     public class HoverLight : MonoBehaviour
     {
-        [SerializeField]
-        [Range(0.0f, 1.0f)]
-        private float radius = 0.15f;
-
-        [SerializeField]
-        private Color color = new Color(0.3f, 0.3f, 0.3f, 1.0f);
-
         // Three hover lights are supported at this time.
         private const int hoverLightCount = 3;
         private const int hoverLightDataSize = 2;
-        private static List<HoverLight> activeHoverLights = new List<HoverLight>(hoverLightCount);
-        private static Vector4[] hoverLightData = new Vector4[hoverLightCount * hoverLightDataSize];
-        private static int _HoverLightDataID;
-        private static int lastHoverLightUpdate = -1;
+        private List<HoverLight> activeHoverLights = new List<HoverLight>(hoverLightCount);
```

Comment:
Sorry if my other comment was confusing. These member variables still need to be static but the member functions do not.

Reply:
No worries then, I'll revert those changes.

Reply:
Thank you!